src/Common/src/Interop/Unix/libssl/Interop.libssl_types.cs

```diff
@@ -86,6 +77,19 @@ static SslMethods()
 #endif
         }
 
+        internal static class CipherString
+        {
+            private const string SSL_TXT_eNULL = "eNULL";
+            private const string SSL_TXT_ALL = "ALL";
+
+            internal const string ALLExceptNULL = SSL_TXT_ALL;
```

Comment:
This name is neither an OpenSSL-defined value (where the style guide says "match native name"), nor conforming to naming rules.

`AllExceptNull` is a much more C#y name.  And `ALL` => `All` or `AllIncludingNull`